src/helpers/inputValidationHelpers.js

```diff
@@ -0,0 +1,30 @@
+import * as constants from '../constants/InputValidation.constants.js';
+
+export const validateName = (value) => {
+    console.log(value);
+    if (!value) {
+        return constants.REQUIRED_FIELD;
+    } else {
+        return null;
+    }
+}
+
+export const validatePassword = (value) => {
+    console.log(value);
+    if (!value) {
+        return constants.REQUIRED_FIELD;
+    } else {
+        return null;
+    }
+}
+
```

Comment:
ValidateName and ValidatePassword are identical functions, maybe we should have only one instead of two?

Reply:
Well, it's more of a placeholder functionality atm